### ISSUE TITLE
`show_palette` broken

### ISSUE THREAD

**Issue by dmarx** [pytti-tools/pytti-core#81]
via Lightnex on discord:

getting an error when enabling ''show palette''

```
TypeError                                 Traceback (most recent call last)
<ipython-input-27-b83fac8072d0> in <module>()
      5 
      6 # function wraps step 2.3 of the original p5 notebook
----> 7 render_frames(cfg)

5 frames
/usr/local/lib/python3.7/dist-packages/hydra/main.py in decorated_main(cfg_passthrough)
     41         def decorated_main(cfg_passthrough: Optional[DictConfig] = None) -> Any:
     42             if cfg_passthrough is not None:
---> 43                 return task_function(cfg_passthrough)
     44             else:
     45                 args = get_args_parser()

/usr/local/lib/python3.7/dist-packages/pytti/workhorse.py in _main(cfg)
    584                 torch.cuda.empty_cache()
    585         else:
--> 586             do_run()
    587             logger.info("Complete.")
    588             gc.collect()

/usr/local/lib/python3.7/dist-packages/pytti/workhorse.py in do_run()
    540                 i_offset=i,
    541                 skipped_steps=skip_steps,
--> 542                 gradient_accumulation_steps=params.gradient_accumulation_steps,
    543             )
    544             skip_steps = 0

/usr/local/lib/python3.7/dist-packages/pytti/ImageGuide.py in run_steps(self, n_steps, prompts, interp_prompts, loss_augs, stop, interp_steps, i_offset, skipped_steps, gradient_accumulation_steps)
    150             #        if appropriate
    151             if not self.null_update:
--> 152                 self.update(i + i_offset, i + skipped_steps)
    153             losses = self.train(
    154                 i + skipped_steps,

/usr/local/lib/python3.7/dist-packages/pytti/ImageGuide.py in update(self, i, stage_i)
    486             display_scale=params.display_scale,
    487             show_graphs=params.show_graphs,
--> 488             show_palette=params.show_palette,
    489         )
    490 

/usr/local/lib/python3.7/dist-packages/pytti/ImageGuide.py in report_out(self, i, stage_i, writer, fig, axs, clear_every, display_every, approximate_vram_usage, display_scale, show_graphs, show_palette)
    389                     im.resize((display_width, display_height), Image.LANCZOS)
    390                 )
--> 391             if show_palette and isinstance(img, PixelImage):
    392                 logger.debug("Palette:")
    393                 display.display(img.render_pallet())

TypeError: isinstance() arg 2 must be a type or tuple of types
```

**Comment by dmarx** [comment]
suspect this was actually addressed by fix for #82 . Bumped reporting user, awaiting feedback to close.

**Comment by karasuuuuu** [comment]
TypeError                                 Traceback (most recent call last)
Input In [87], in <cell line: 3>()
      1 from pytti.workhorse import _main as render_frames
----> 3 render_frames(cfg)

File D:\GAN\envs\pytti-vid\lib\site-packages\hydra\main.py:43, in main.<locals>.main_decorator.<locals>.decorated_main(cfg_passthrough)
     40 @functools.wraps(task_function)
     41 def decorated_main(cfg_passthrough: Optional[DictConfig] = None) -> Any:
     42     if cfg_passthrough is not None:
---> 43         return task_function(cfg_passthrough)
     44     else:
     45         args = get_args_parser()

File D:\GAN\envs\pytti-vid\lib\site-packages\pytti\workhorse.py:588, in _main(cfg)
    586         torch.cuda.empty_cache()
    587 else:
--> 588     do_run()
    589     logger.info("Complete.")
    590     gc.collect()

File D:\GAN\envs\pytti-vid\lib\site-packages\pytti\workhorse.py:536, in _main.<locals>.do_run()
    534 for scene in prompts[skip_prompts:]:
    535     logger.info("Running prompt:", " | ".join(map(str, scene)))
--> 536     i += model.run_steps(
    537         params.steps_per_scene - skip_steps,
    538         scene,
    539         last_scene,
    540         loss_augs,
    541         interp_steps=params.interpolation_steps,
    542         i_offset=i,
    543         skipped_steps=skip_steps,
    544         gradient_accumulation_steps=params.gradient_accumulation_steps,
    545     )
    546     skip_steps = 0
    547     model.clear_dataframe()

File D:\GAN\envs\pytti-vid\lib\site-packages\pytti\ImageGuide.py:152, in DirectImageGuide.run_steps(self, n_steps, prompts, interp_prompts, loss_augs, stop, interp_steps, i_offset, skipped_steps, gradient_accumulation_steps)
    143 for i in tqdm(range(n_steps)):
    144     # not a huge fan of this.
    145     # currently need it for PixelImage.encode_image
   (...)
    149     #        initialized with a renderer or not, and call self.renderer.update()
    150     #        if appropriate
    151     if not self.null_update:
--> 152         self.update(i + i_offset, i + skipped_steps)
    153     losses = self.train(
    154         i + skipped_steps,
    155         prompts,
   (...)
    159         gradient_accumulation_steps=gradient_accumulation_steps,
    160     )
    161     if losses["TOTAL"] <= stop:

File D:\GAN\envs\pytti-vid\lib\site-packages\pytti\ImageGuide.py:476, in DirectImageGuide.update(self, i, stage_i)
    473 img = self.image_rep
    474 embedder = self.embedder
--> 476 model.report_out(
    477     i=i,
    478     stage_i=stage_i,
    479     # model=model,
    480     writer=writer,
    481     fig=fig,  # default to None...
    482     axs=axs,  # default to None...
    483     clear_every=params.clear_every,
    484     display_every=params.display_every,
    485     approximate_vram_usage=params.approximate_vram_usage,
    486     display_scale=params.display_scale,
    487     show_graphs=params.show_graphs,
    488     show_palette=params.show_palette,
    489 )
    491 model.save_out(
    492     i=i,
    493     # img=img,
   (...)
    499     backups=params.backups,
    500 )
    502 # animate
    503 ################
    504 ## TO DO: attach T as a class attribute

File D:\GAN\envs\pytti-vid\lib\site-packages\pytti\ImageGuide.py:391, in DirectImageGuide.report_out(self, i, stage_i, writer, fig, axs, clear_every, display_every, approximate_vram_usage, display_scale, show_graphs, show_palette)
    387     im = img.decode_image()
    388     display.display(
    389         im.resize((display_width, display_height), Image.LANCZOS)
    390     )
--> 391 if show_palette and isinstance(img, PixelImage):
    392     logger.debug("Palette:")
    393     display.display(img.render_pallet())

TypeError: isinstance() arg 2 must be a type or tuple of types
